plugins/flutter_aepconsent/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
I will refer to the license in the root folder.  Let me discuss with @cdhoffmann about taking out the license from each plugin.

Reply:
@emdobrin @cacheung The license is needed in each plugin directory because Flutter checks for a license in the plugin directory when you publish the plugin.

Reply:
I am keeping the license in the consent directory.  Emilia points out we need to change the license to year 2022.  @cdhoffmann Do you want me to update the year for all the extensions license in this PR? Or want to do it later?

Reply:
@cdhoffmann Should I change the Copyright year for Core extensions? Currently, they have year 2020.